lib/timber.ex

```diff
@@ -86,40 +89,54 @@ defmodule Timber do
     delete_context(key, location)
   end
 
-  @doc """
-  Used to time runtime execution. For example, when timing a `Timber.Events.HTTPResponseEvent`:
+  @doc ~S"""
+  Used to time runtime execution.
+
+  We highly recommend using this method as it uses the system monotonic time for
+  accuracy.
 
-  ```elixir
-  timer = Timber.start_timer()
-  # .... make request
-  time_ms = Timber.duration_ms(timer)
-  event = HTTPResponseEvent.new(status: 200, time_ms: time_ms)
-  message = HTTPResponseEvent.message(event)
-  Logger.info(message, event: event)
-  ```
+  ## Example
+
+      timer = Timber.start_timer()
+      # .... do something
+      duration_ms = Timber.duration_ms(timer)
+      event = %{job_completed: %{duration_ms: duration_ms}}
+      message = "Job completed in #{duration_ms}ms"
+      Logger.info(message, event: event)
 
   """
-  defdelegate start_timer, to: Timber.Timer, as: :start
+  defdelegate start_timer,
+    to: Timber.Timer,
+    as: :start
 
   #
   # Utilility methods
+  #
   # The following methods are used for internally throughout Timber and it's
   # dependent integration libraries.
   #
 
+  # This method should be used for logging internal events.
+  #
+  # Because Timber is a logger it cannot log like a traditional library. This will create
+  # a loop of debug messages. Instead we write to a configurable IO device that the user can
+  # inspect, such as `STDOUT`, `STDERR`, a file.
   @doc false
-  def debug(message_fun) do
+  def log(level, message_fun) do
```

Comment:
should the doc include example configuration?
something like:
```elixir
config :timber, debug_io_device: :stdio
```

Reply:
Yep, just updated